docs/api/rest_api_reference/input/ez-user-groups.raml

```diff
@@ -3,7 +3,7 @@ get:
     description: Loads User Groups for either an an ID or a remote ID or a Role.
     queryParameters:
         roleId:
-            description: Lists User Groups assigned to the given Role (e.g. GET /user/users?roleId=/user/roles/1).
+            description: Lists User Groups assigned to the given Role (e.g. GET /user/groups?roleId=/user/roles/1).
```

Comment:
/user/users and /user/groups don't handle the roleId parameter the same way.
- /user/users parses it as an href: https://github.com/ibexa/rest/blob/main/src/lib/Server/Controller/User.php#L465
- /user/groups uses it directly as an ID: https://github.com/ibexa/rest/blob/main/src/lib/Server/Controller/User.php#L572

```suggestion
            description: Lists User Groups assigned to the given Role (e.g. GET /user/groups?roleId=1).
```